order/apps.py

```diff
@@ -4,3 +4,6 @@
 class OrderConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'order'
+
+    def ready(self):
+        import order.signals
```

Comment:
why?

Reply:
the signals do not work without this code